ui/partials/perf/comparechooserctrl.html

```diff
@@ -9,17 +9,21 @@
         <div class="panel-body">
           <label for="original-project-selector">Project</label>
           <select id="original-project-selector" class="form-control" ng-model="originalProject" ng-options="project.name for project in projects"  ng-change="updateOriginalRevisionTips()"/>
-          <label for="original-revision-input">Revision</label>
-          <div class="form-group" ng-class="{'has-error': originalRevisionError}">
-            <input id="original-revision-input" maxlength="40" class="form-control" type="text" ng-model="originalRevision" placeholder="Select or enter a revision"/>
-            <div class="input-group-btn">
-              <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-              Recent <span class="caret"></span></button>
-              <ul class="dropdown-menu dropdown-menu-right" ng-model="tipRevision">
-                <li ng-repeat="tip in originalTipList"><a ng-click="getOriginalTipRevision(tip.revision)">{{tip.revision | limitTo: 12}} {{tip.author}}</a></li>
+          <label><input type="checkbox" ng-model="baseLineComparison"> Base Line Comparison</label><br>
```

Comment:
I would put this inside a div instead of using `<br>`

Follow the example here: http://getbootstrap.com/css/#forms